docs/var.tfvars-doc.md

```diff
@@ -42,23 +42,25 @@ These set of variables specify the cluster capacity.
 ```
 bastion                     = {instance_type    = "<bastion-compute-template>", image_id    = "<image-uuid-rhel>",  "count"   = 1}
 bootstrap                   = {instance_type    = "<bootstrap-compute-template>", image_id    = "<image-uuid-rhcos>",  "count"   = 1}
-master                      = {instance_type    = "<master-compute-template>", image_id    = "<image-uuid-rhcos>",  "count"   = 3}
-worker                      = {instance_type    = "<worker-compute-template>", image_id    = "<image-uuid-rhcos>",  "count"   = 2}
+master                      = {instance_type    = "<master-compute-template>", image_id    = "<image-uuid-rhcos>",  "count"   = 3, "data_volume_size"  = "100", "data_volume_count"  = "0"}
+worker                      = {instance_type    = "<worker-compute-template>", image_id    = "<image-uuid-rhcos>",  "count"   = 2, "data_volume_size"  = "100", "data_volume_count"  = "0"}
 ```
 
-`instance_type` is the compute template to be used and `image_id` is the image UUID. `count` specifies the number of VMs that should be created for each type.
+`instance_type` is the compute template to be used and `image_id` is the image UUID. `count` specifies the number of VMs that should be created for each type. `data_volume_size` is the default volume size to be attached to the nodes. `data_volume_count` is the number of volumes to be attached to each node.
```

Comment:
1. Can we have `data_volume_count` preceding `data_volume_size`? Also mention these are optional.
2. Better to move this from the default configurations. See how it is described for providing availability zone. A new para will help avoiding confusions for users who don't want to use this feature.

Reply:
Now I see down that you have added these variables to all examples. Please use an exclusive para for these, as they are not required.
Thanks.